fuzzers/100-dsp-mskpat/generate.py

```diff
@@ -1,23 +1,134 @@
 #!/usr/bin/env python3
 
 from prjxray.segmaker import Segmaker
-import csv
+from prjxray.verilog import to_int
+from prjxray.verilog import quote
+import json
 
-segmk = Segmaker("design.bits", verbose=True)
 
-print("Loading tags")
-with open('params.csv', 'r') as f:
-    for d in csv.DictReader(f):
-        dsp = "DSP_0" if d['site'][-1] in "02468" else "DSP_1"
+def bits_in(value, width):
```

Comment:
This looks like a int to bitarray function?  Pick a better name, and add a function comment.